packages/backend/src/api/endpoints/SamlConnectionApi.ts

```diff
@@ -3,13 +3,18 @@ import type { SamlIdpSlug } from '@clerk/types';
 import { joinPaths } from '../../util/path';
 import type { SamlConnection } from '../resources';
 import { AbstractAPI } from './AbstractApi';
+import type { WithSign } from './util-types';
 
 const basePath = '/saml_connections';
 
 type SamlConnectionListParams = {
   limit?: number;
   offset?: number;
+  query?: string;
+  orderBy?: WithSign<'phone_number' | 'email_address' | 'created_at' | 'first_name' | 'last_name' | 'username'>;
+  organizationId?: WithSign<string>[];
```

Comment:
I don't believe we need `WithSign` here:

```suggestion
  organizationId?: string[];
```

Reply:
I'm happy to remove it, the documentation just made it seem like this was supported:

> the `+` and `-` can be prepended to the id, which denote whether the respective organization should be included or excluded from the result set